console/client/src/layout/IDELayout.tsx

```diff
@@ -121,7 +133,7 @@ export function IDELayout() {
               </Tab.List>
               <div className='flex-grow'></div>
             </div>
-            <div className={`flex-1 overflow-y-scroll ${panelColor}`}>
+            <div className={`flex-1 overflow-y-scroll ${panelColor} p-4`}>
```

Comment:
This `p-4` adds extra padding to the timeline view which causes items to appear above the bar
<img width="1096" alt="Screenshot 2023-08-25 at 9 22 12 AM" src="https://github.com/TBD54566975/ftl/assets/51647/4b85fee8-8376-4ee4-9016-2cb43cd09e0c">

Main currently looks like this:
<img width="1025" alt="Screenshot 2023-08-25 at 9 25 53 AM" src="https://github.com/TBD54566975/ftl/assets/51647/042c42e9-8746-4196-a0a8-bcc2b7118e99">